x-pack/plugins/kubernetes_security/common/translations.ts

```diff
@@ -7,10 +7,6 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const BETA = i18n.translate('xpack.kubernetesSecurity.beta', {
```

Comment:
Same here: we need to leave the Beta tag on until we can bring the Defend for Containers integration to GA.